home-data/src/main/java/com/softserveinc/ita/homeproject/homedata/entity/InvitationStatusAttributeConverter.java

```diff
@@ -0,0 +1,49 @@
+package com.softserveinc.ita.homeproject.homedata.entity;
+
+import javax.persistence.AttributeConverter;
+
+public class InvitationStatusAttributeConverter implements AttributeConverter<InvitationStatus, String> {
+    @Override
+    public String convertToDatabaseColumn(InvitationStatus attribute) {
+        if (attribute == null) {
+            return null;
+        }
+
+        switch (attribute) {
+            case PENDING:
```

Comment:
Use enum like attribute.toString()

Reply:
we will open new issue to refactor attributeConverter to generic converter